### DISCUSSION TITLE
정원이 차지 않았음에도 새로운 방이 생성되는 문제

### DISCUSSION THREAD

**Discussion by esang28** [naverz/zepeto-studio-kor#997] [5. Q&A]
안녕하세요, 
월드 심사에 통과하고 출시가 된 월드를 관리하고 있습니다.

타 월드에서는 입장하기 버튼을 누르면 누군가 이미 생성해 놓은 방으로 입장이 되어지는데, 
제가 관리하는 월드에서는 기존에 누가 방을 생성을 해두고 정원이 차지 않았음에도 각자 개별의 방을 생성하는 것 같습니다.

입장하기 버튼을 통해서 입장할 때, 정원이 차지 않은 방으로 먼저 이동할 수 있게 하는 방법은 어디에서 건드리면 좋을까요?

**Comment by YSHFM** [comment]
저도 테스트 링크 이용 시에 각자 개별로 룸이 생성 되는 것 같은데 동일한 증상인 있네요....

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. https://github.com/naverz/zepeto-studio-kor/discussions/567 이슈와 동일한 내용으로 개선 작업 진행 중에 있습니다. 
감사합니다.

**Reply by esang28** [reply]
테스트 링크가 아닌 이미 심사를 거쳐 출시된 월드를, 제페토 앱에서 입장할 때 정원인원에 상관없이 개별의 룸이 생긴다는 내용이었습니다. 이것 또한 링크하신 페이지와 관련이 있는 문제일까요?

**Reply by Zepeto-TechEvangelist** [reply]
네, 심사 중인 월드 역시 실제 제페토 서버에서 만나는 구조입니다.
동일한 월드에 접속할 경우 열려있는 방에 몰아주지 않고 개별로 입장하는 부분이 개선된다면 두 이슈가 원하는 부분이 충족될 것입니다.
